sky/packages/sky/lib/src/rendering/paragraph.dart

```diff
@@ -59,22 +59,27 @@ class RenderParagraph extends RenderBox {
     textPainter.minHeight = constraints.minHeight;
     textPainter.maxHeight = constraints.maxHeight;
     textPainter.layout();
+    // By default, we shrinkwrap to the intrinsic width.
+    double width = constraints.constrainWidth(textPainter.maxIntrinsicWidth);
```

Comment:
Aah, I see. so we didn't end up laying the text out at our width. Makes sense. Thanks.